### PR TITLE
Add requirements to setup.py

### LINKED ISSUE #108: Specify dependencies in setup.py
We should specify the install_requires, similar to https://github.com/mozilla/libmozdata/blob/master/setup.py#L30.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #108

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,5 +7,6 @@
     name='coverage_crawler',
     version='1.0.0',
     description='A crawler to find websites that exercise code in Firefox that is not covered by unit tests',
+    install_requires=load_requirements('requirements.txt'),
```
Comment (marco-c) [inline setup.py]: You'll also need to add the load_requirements function :)

Comment (rhcu) [reply setup.py]: I  just noticed. Sorry for that :)

Comment (marco-c) [reply setup.py]: Ah, it looks like setup.py can't handle requirements like the ones starting with "-e", so this simple `load_requirements` is not enough.

You could use the more complex one from https://github.com/mozilla/release-services/blob/13285250a9bdfdac1b4b6c04e4c897dac372d462/src/shipit_code_coverage_backend/setup.py#L15.

File: setup.py [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+
+import os
+
 from setuptools import find_packages
 from setuptools import setup
 
+
+def read_requirements(file_):
+    lines = []
+    with open(os.path.join(os.path.dirname(__file__), file_)) as f:
```
Comment (marco-c) [inline setup.py]: Maybe you have to move this outside the function, like https://github.com/mozilla/libmozdata/blob/master/setup.py#L12.

File: setup.py [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
+            lines.append(line)
+    return sorted(list(set(lines)))
+
+
 setup(
     name='coverage_crawler',
     version='1.0.0',
     description='A crawler to find websites that exercise code in Firefox that is not covered by unit tests',
+    install_requires=read_requirements(filename),
     packages=find_packages(exclude=['contrib', 'docs', 'tests'])
```
Comment (marco-c) [inline setup.py]: Maybe you need to add `include_package_data=True` and/or `zip_safe=False`.

**Comment by rhcu** [comment]
Travis cannot find `requirements.txt` but it works locally. :(

**Comment by marco-c** [comment]
> Travis cannot find requirements.txt but it works locally. :(

I can reproduce the error, I'll try to fix it.

**Comment by marco-c** [comment]
I've fixed the problem in https://github.com/mozilla/coverage-crawler/pull/114.

**Pull request closed without merging**